### PR TITLE
Issue: Increased geolocation accuracy in Excel export (SP 1) #278

### PR DESCRIPTION
**Pull request by webaissance**
Fix for #278
* Add header for Latitude, Longitude, Accuracy
* Add export fields for latitude, longitude, accuracy

### CODE REVIEW

File: app/models/submission.rb [hunk 1]
```diff
@@ -281,14 +281,14 @@ def self.get_satisfaction(satisfaction)
...
-    'Response #', 'Source', 'Date', 'How Are You Testing', 'Zip', 'Census Tract', 'Census Block',
+    'Response #', 'Source', 'Date', 'How Are You Testing', 'Zip', 'Census Tract', 'Census Block', 'Latitude', 'Longitude', 'Accuracy',
     'Provider', 'How are you connected', 'Price Per Month', 'Advertised Download Speed',
     'Satisfaction Rating', 'Download Speed', 'Upload Speed', 'Advertised Price Per Mbps',
     'Actual Price Per Mbps', 'Ping'
   ]
 
   CSV_KEYS = [
-    :id, :source, :date, :testing_for, :zip_code, :census_code, :census_block, :provider, :connected_with,
+    :id, :source, :date, :testing_for, :zip_code, :census_code, :census_block, :latitude, :longitude, :accuracy, :provider, :provider, :connected_with,
```
Comment (noahbrenner) [inline app/models/submission.rb:291]: Would you remove the doubled `:provider` in here? That only needs to be included once.

**Review by noahbrenner — CHANGES_REQUESTED** [review]
Thanks for contributing, @webaissance! Just one little adjustment to be made before we can merge this in:

File: app/models/submission.rb (hunk 1 above)
Comment (webaissance) [reply app/models/submission.rb]: Thanks Noah. Silly mistake on my part. I fixed it just now.

**Review by noahbrenner — APPROVED** [review]
Looks good!

**Pull request merged**